README.md

```diff
@@ -316,11 +314,97 @@ it('should use custom element', async() => {
 
 > **Important:** `defineCE` is not performant and shouldn't be used outside of test files.
 
-### Vdiff Testing
+## Running Tests
+
+Use the `d2l-test-runner` binary to execute a set of tests and report their results. It builds upon the robust [@web/test-runner](https://modern-web.dev/docs/test-runner/overview/), while configuring it for Brightspace components and applications.
+
+### CLI and Configuration
+
+`d2l-test-runner` can be configured using CLI flags or an optional configuration file.
 
-Short for "visual diff" and also known as "visual regression" or "perceptual diff", vdiff testing involves taking snapshot images of the browser and comparing them against a known golden (or "baseline") image. The images are compared pixel-by-pixel and differences beyond a threshold will fail the test. Our vdiff testing leverages the [pixelmatch](https://github.com/mapbox/pixelmatch) library to perfom its comparison.
+#### Options
 
-Use the asynchronous `.to.be.golden()` Chai assertion to take a vdiff snapshot and compare it against its golden.
+| Name | Type | Default | Description |
+| :--- | :---: | :---: | :--- |
+| group | `String` | `'test'` | Name of the test group to run |
+| chrome | `Boolean` | `true` | Run tests in Chromium |
+| firefox | `Boolean` | `true` | Run tests in Firefox |
+| safari | `Boolean` | `true` | Run tests in Webkit |
+| timeout | `Number` | `2000` | Test timeout threshold in ms |
+| filter | `String` | | Filter test files by replacing wildcards with this glob |
+| grep | `String` | | Only run tests matching this string or regexp |
+| files | `String` | `'./test/**/*.<group>.js'` | Test files to run. Path or glob. |
+| config | `String` | `'./d2l-test-runner.config.js'` | Location of config file |
+| watch | `Boolean` | `false` | Reload tests on file changes. Allows debugging in all browsers. |
+| open | `Boolean` | `false` | Open the browser in headed mode |
+| slowmo | `Number` | | Slows down test operations by the specified number of milliseconds. Useful for debugging. |
+| slow | `Number` | `75` | Tests whose duration in milliseconds are at most half of this threshold are "fast" and tests which exceed it are "slow" |
+
+For example, to run all tests in the default `'test'` group in Firefox and with a `3s` timeout:
+
+```bash
+d2l-test-runner --firefox --timeout=3000
```

Comment:
I always do this:
```suggestion
d2l-test-runner --firefox --timeout 3000
```
Are they equivalent?

Reply:
They both work for me so I think they are equivalent, but I can remove the `=` if we think that syntax is more common.